tutor/resources/styles/components/student-dashboard/task.less

```diff
@@ -41,6 +41,15 @@
         color: @tutor-danger;
       }
     }
+    .btn.-hide-button {
+      padding-left: 15px;
```

Comment:
Hehe, the `-` prefix is intended to indicate that the class isn't intended for styling.  They're for selection during testing.  No reason you would have know that though - I'll remove the `-` prefix :)